base_layer/wallet_ffi/src/lib.rs

```diff
@@ -5259,6 +5259,7 @@ pub unsafe extern "C" fn wallet_create(
     passphrase: *const c_char,
     seed_words: *const TariSeedWords,
     network_str: *const c_char,
+    peer_seed_str: *const c_char,
```

Comment:
I don't think we need a whole parameter for this. It's based on the network, and we already have that:

```
dns_seeds: StringList::from(vec![format!("seeds.{}.tari.com", network.as_key_str())])
```

Reply:
thats only if we dont change it...